docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '2'
+services:
+  spark-master:
+    build: .
+    command: /usr/local/bootstrap.sh master
+    ports:
+      - "8080:8080"
+      - "7077:7077"
+      - "6066:6066"
+    environment:
+      - SPARK_MASTER_HOST=spark-master
+  spark-worker:
+    build: .
+    command: /usr/local/bootstrap.sh worker
+    links:
```

Comment:
@kalmanchapman let's open the worker webui port as well for introspection. I believe it's 4040 but check me on that.